docs/app/pages/components/sections/input-controls/sliders/sliders.component.html

```diff
@@ -77,6 +77,8 @@ <h4 class="text-black p-t-md p-b-md">Range Slider with Text Inputs</h4>
 <p>The slider component can be used to allow a user to select a single value or a range of values. It can easily be customized
     to match the desired appearance and behavior.</p>
 
+<br>
+
```

Comment:
Use CSS to adjust spacing above headers, so that it is consistent between all sections and can be adjusted.